pkgs/development/libraries/libnbd/default.nix

```diff
@@ -4,6 +4,7 @@
 , bash-completion
 , pkg-config
 , perl
+, withPython ? true, python3
```

Comment:
This should by default be false because we don't need the binding in the normal package, do we?

Reply:
I'm honestly on the fence with this one. I've searched for similar options in other packages and it seems to be fairly 50-50 whether the default is off or on. I guess the counterpoint is that defaulting to `true` means you have one fewer copy of the derivation floating around in the cache?

Reply:
I think Nixpkgs will be smart enough to not to do useless duplication.